src/components/list_group/list_group_item.tsx

```diff
@@ -330,6 +336,7 @@ export const EuiListGroupItem: FunctionComponent<EuiListGroupItemProps> = ({
           content={toolTipText ?? label}
           position="right"
           delay="long"
+          {...toolTipProps}
```

Comment:
We need to ensure `anchorClassName` and `anchorProps` merge with (instead of being overridden by) consumers' `toolTipProps`:

```suggestion
          {...toolTipProps}
          anchorClassName={classNames('euiListGroupItem__tooltip', toolTipProps?.anchorClassName)}
          anchorProps={{ ...toolTipProps?.anchorProps, css: [cssTooltipStyles, tooltTipProps?.anchorProps?.css] }}
```

The above code is not perfectly formatted code, please feel free to DRY stuff out how you think looks best.